applications/asset_tracker_v2/src/cloud/lwm2m_integration/configuration_object.c

```diff
@@ -26,11 +26,13 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #define RESOURCE_ACTIVE_WAIT_TIMEOUT		2
 #define RESOURCE_MOVEMENT_RESOLUTION		3
 #define RESOURCE_MOVEMENT_TIMEOUT		4
-#define RESOURCE_ACCELEROMETER_THRESHOLD	5
-#define RESOURCE_GNSS_ENABLE			6
-#define RESOURCE_NEIGHBOR_CELL_ENABLE		7
+#define RESOURCE_ACCELEROMETER_ACT_THRESHOLD	5
+#define RESOURCE_ACCELEROMETER_INACT_THRESHOLD	6
+#define RESOURCE_ACCELEROMETER_INACT_TIMEOUT	7
+#define RESOURCE_GNSS_ENABLE			8
+#define RESOURCE_NEIGHBOR_CELL_ENABLE		9
```

Comment:
As mentioned in the other comment, please append to the existing list and not modify RIDs of existing parameters.